packages/p2p-v2/src/pages/buy-sell/screens/BuySellHeader/BuySellHeader.tsx

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
+import clsx from 'clsx';
+import { TSortByValues } from '@/utils';
 import { Tab, Tabs } from '@deriv-com/ui';
+import { SortDropdown } from '../../components';
 import './BuySellHeader.scss';
 
 type TBuySellHeaderProps = {
     activeTab: string;
     setActiveTab: (tab: string) => void;
+    setIsFilterModalOpen: (value: boolean) => void;
+    setSortDropdownValue: (value: TSortByValues) => void;
+    sortDropdownValue: TSortByValues;
 };
 
-const BuySellHeader = ({ activeTab, setActiveTab }: TBuySellHeaderProps) => {
+const sortDropdownList = [
```

Comment:
can we use the constant here?

Reply:
ah yeah can replace it with the one i made